### DISCUSSION TITLE
Upll clock

### DISCUSSION THREAD

**Discussion by CEmir10** [zephyrproject-rtos/zephyr#49080] [Q&A] [answered]
i used can driver which used plla clock and i was able to send message with low bitrate, when i switched to can driver that uses upll clock i am getting error "can_mcan can not leave init mode". I am using same70j19 custom board. Can you help me to fix this problem?

**Comment by henrikbrixandersen** [comment, marked as answer]
You already asked this in https://github.com/zephyrproject-rtos/zephyr/issues/45012#issuecomment-1174814952 and received a reply in https://github.com/zephyrproject-rtos/zephyr/issues/45012#issuecomment-1175011333.

**Comment by CEmir10** [comment]
Sorry, but this is not an answer, i have a Crystal oscillator, but unfortunately it is not working for me. I am wondering if i was able to use Driver with plla Clock, why i can not use the one with upll Clock.

Von: Henrik Brix ***@***.***>
Gesendet: Montag, 15. August 2022 21:22
An: ***@***.***>
Cc: ***@***.***>; ***@***.***>
Betreff: Re: [zephyrproject-rtos/zephyr] Upll clock (Discussion #49080)


You already asked this in #45012 (comment)<https://github.com/zephyrproject-rtos/zephyr/issues/45012#issuecomment-1174814952> and received a reply in #45012 (comment)<https://github.com/zephyrproject-rtos/zephyr/issues/45012#issuecomment-1175011333>.
